crates/hotshot/src/tasks/mod.rs

```diff
@@ -407,6 +408,78 @@ where
     task_runner.add_task(da_task_id, da_name.to_string(), da_task)
 }
 
+/// add the Transaction Handling task
+/// # Panics
+/// Is unable to panic. This section here is just to satisfy clippy
+pub async fn add_transaction_task<
+    TYPES: NodeType,
+    I: NodeImplementation<
+        TYPES,
+        Leaf = SequencingLeaf<TYPES>,
+        ConsensusMessage = SequencingMessage<TYPES, I>,
+    >,
+>(
+    task_runner: TaskRunner,
+    event_stream: ChannelStream<SequencingHotShotEvent<TYPES, I>>,
+    committee_exchange: CommitteeEx<TYPES, I>,
+    handle: SystemContextHandle<TYPES, I>,
+) -> TaskRunner
+where
+    CommitteeEx<TYPES, I>: ConsensusExchange<
+        TYPES,
+        Message<TYPES, I>,
+        Certificate = DACertificate<TYPES>,
+        Commitment = TYPES::BlockType,
+    >,
+{
+    // build the da task
```

Comment:
Nit: `da` -> `transaction`.

Reply:
Or `transactions`? Looks like we use the singular form in struct names, but the plural form in comments.